app/src/main/java/com/futurice/freesound/feature/search/SearchViewModel.java

```diff
@@ -56,7 +58,8 @@
     private final Analytics analytics;
 
     @NonNull
-    private final BehaviorRelay<String> searchTermRelay = BehaviorRelay.create(NO_SEARCH);
+    private final BehaviorSubject<String> searchTermRelay = BehaviorSubject
```

Comment:
If you declare this as a `Subject<String, String>` it might not need the linewrap. :-)